### DISCUSSION TITLE
Page Preview and Layer Preview Backgrounds Need a Dark Mode

### DISCUSSION THREAD

**Discussion by K4LCIFER** [xournalpp/xournalpp#2644] [Ideas]
When you select "Dark Mode" the background of the "page  preview" and "layer preview" panes remains pretty jarringly bright, but it should follow the dark mode theme the same as the "contents" pane.

**Comment by LittleHuba** [comment]
The "Dark Mode" is not actually a dark theme. It is only meant to provide light buttons for users that use a dark theme. Hence you would need to switch your theme to achieve dark backgrounds in those views.

**Comment by personalizedrefrigerator** [comment]
There's an open PR that should address this! See https://github.com/xournalpp/xournalpp/pull/2726#issue-564355317 

**Edit:** Clarification
